src/metadata.rs

```diff
@@ -20,7 +41,32 @@ fn get_metadata_file(root_path: &str, id: &str) -> io::Result<PathBuf> {
     };
 }
 
-pub fn get_metadata(root_path: &str, id: &str) -> io::Result<serde_json::Value> {
+pub fn get_metadata_from_date(root_path: &str, from: Option<String>) -> io::Result<Vec<Packet>> {
+    let path = Path::new(root_path)
+        .join(".outpack")
+        .join("metadata");
+
+    validate_datetime(&from)?;
+
+    let packets = fs::read_dir(path)?
+        .filter_map(|e| e.ok())
+        .filter(|e| utils::is_packet(&e.file_name()));
+
+    let mut packets = match from {
+        None => packets.map(|entry| read_entry(entry.path()))
+            .collect::<io::Result<Vec<Packet>>>()?,
+        Some(id) => {
+            packets.filter(|entry| entry.file_name().into_string().unwrap() > id)
```

Comment:
So, I think that this is not correct, though it will often do the right thing. The date that we really care about is the date that the packet was inserted into the index probably? For that, you want the `time` element of the location metadata (https://github.com/mrc-ide/outpack/blob/main/inst/schema/location.json#L19C10-L21).

Practically this has no difference if everyone is running things on the server only. But if people are importing packets, then it's the time in location that will be the one you'd want if you're trying to keep the packet list synchronised